proj4/src/main/java/org/osgeo/proj4j/proj/Projection.java

```diff
@@ -253,7 +253,7 @@ public ProjCoordinate projectRadians( ProjCoordinate src, ProjCoordinate dst ) {
      */
     private ProjCoordinate projectRadians(double x, double y, ProjCoordinate dst ) {
         project(x, y, dst);
-        if (unit == Units.DEGREES) {
+        if (unit != null && unit.equals(Units.DEGREES)) {
```

Comment:
@dwins just as an FYI, there were 2 other places in this file (this is one) where there were serialization issues w/ reference equaliy.